recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.7.1" %}
+{% set md5 = "c47b36ceb17eaff3ecfab3bcd347d0df" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Basically correct.

Obviously `script:` will be dropped.

Also some people prefer to use the environment variable for `python`. So that would be `$PYTHON` on Unix and `%PYTHON%` on Windows.

Finally we need to delete the versioned entrypoints, but we can deal with that after we break this out.